app/models/course.rb

```diff
@@ -442,7 +442,7 @@ def gcse_science_required?
   end
 
   def gcse_grade_required
-    if PROVIDERS_REQUIRING_GCSE_GRADE_5.include?(provider_code)
+    if PROVIDERS_REQUIRING_GCSE_GRADE_5.include?(provider_code) || PROVIDERS_REQUIRING_GCSE_GRADE_5.include?(accrediting_provider&.provider_code)
```

Comment:
Could potentially use `#any?` to make this more succinct?

Reply:
have made the change